libraries/AP_Rally/AP_Rally.h

```diff
@@ -53,6 +53,7 @@ class AP_Rally {
 
     // data handling
     bool get_rally_point_with_index(uint8_t i, RallyLocation &ret) const;
+    bool get_rally_point_with_index(uint8_t i, Location &ret) const;
```

Comment:
I think this should be get_rally_location_with_index - this will make Lua bindings easier (Lua will probably want either Location or RallyLocation I'm thinking).